test/Microsoft.AspNetCore.Razor.Test.Common/Language/Legacy/ParserTestBase.cs

```diff
@@ -91,6 +91,8 @@ public static bool IsTheory
 #endif
         }
 
+        protected int BaselineTestCount { get; set; }
```

Comment:
I needed this to support the cases when we need to test multiple baselines within a single test